docs/index.rst

```diff
@@ -161,8 +161,8 @@ of Read the Docs and the larger software documentation ecosystem.
 
 * **Getting involved with Read the Docs**:
   :doc:`Contributing <contribute>` |
+  :doc:`Docker development setup </development/standards>` |
```

Comment:
```suggestion
  :doc:`Development setup </development/standards>` |
```

I don't think there's any need to reference "docker" here. This is just the preferred development setup for working on RTD.